src/lib.rs

```diff
@@ -32,3 +32,6 @@
 
 #[doc(inline)]
 pub use derive_more_impl::*;
+
+mod errors;
+pub use crate::errors::TryIntoError;
```

Comment:
Also, I think this should be feature-gated by `"try_into"`.

Reply:
Done in https://github.com/JelteF/derive_more/pull/220/commits/77043d1fd53825cb09eb3ada86f1e1b11853140f.